customize_image.sh

```diff
@@ -54,11 +56,31 @@ eval set -- "$MYOPTS"
 while true ; do
 	case "$1" in
 		-m|--minimize) MINIMG=1 ; shift ;;
+		-f|--force) FORCE=1 ; shift ;;
 		-h|--help) usage ; exit 0;;
 		*) shift; break ;;
 	esac
 done
 
+SOURCE_IMAGE=$1
+OUTPUT_IMAGE=${2:-$1}
+
+if [ "$SOURCE_IMAGE" == "$OUTPUT_IMAGE" ]; then
+	echo "WARNING: No name for the output image was specified. The source image will be modfied in-place."
+
+	if [ $FORCE -eq 0 ]; then
+		echo ""
+		echo -n "Do you want to continue? [Ny] "
+		read -r choice
+
+		if ! [[ "$choice" == "y" || "$choice" == "y" ]]; then
+			exit
+		fi
+	fi
+else
+	cp "$SOURCE_IMAGE" "$OUTPUT_IMAGE"
```

Comment:
Do somebody know a "verbose" copy command, just to show the user "We are working and alive...". This command could take a "very" long time - would me confuse :D

Reply:
I've added a info message before the copy starts